### PR TITLE
Assignment submission

### PR DESCRIPTION
**Pull request by ryanflach**
Feedback appreciated.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 class User < ActiveRecord::Base
+  has_many :authored_reviews, class_name: "Review", foreign_key: :author_id
```
Comment (case-eee) [inline app/models/user.rb:2]: 👍

File: app/models/vote.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class Vote < ActiveRecord::Base
+  belongs_to :voter, class_name: "User"
+  belongs_to :votable, polymorphic: true
+  validates :voter_id, uniqueness: {scope: [:votable_id, :votable_type], message: "User allowed only one vote per movie/review"}
```
Comment (case-eee) [inline app/models/vote.rb:4]: woooo 🎉

**Comment by case-eee** [comment]
hey @ryanflach - i think this looks really solid. great work 🎉  do you have any questions for me on all these associations?

**Comment by ryanflach** [comment]
Thanks! No questions right now, but I'll probably have some during the mini project. Hoping things stick, but it's a lot of new syntax, so we'll see. Thanks again.

**Comment by case-eee** [comment]
you're welcome! as always - just let me know

**Pull request closed without merging**